packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -550,87 +602,101 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
   const variables: CSSOutput['variables'] = [];
 
   // Quasis are the string pieces of the template literal - the parts around the interpolations
-  const literalResult = node.quasis.reduce<string>((acc, quasi, index): string => {
-    const nodeExpression = node.expressions[index] as t.Expression | undefined;
-
-    if (
-      !nodeExpression ||
-      (t.isArrowFunctionExpression(nodeExpression) && t.isLogicalExpression(nodeExpression.body))
-    ) {
-      const suffix = meta.context === 'keyframes' ? '' : ';';
-      return acc + quasi.value.raw + suffix;
-    }
+  const literalResult = node.quasis.reduce<string>(
+    (acc: string, quasi: t.TemplateLitteral, index: number): string => {
+      const nodeExpression = node.expressions[index] as t.Expression | undefined;
+
+      // Deal with any negative values such as:
+      // margin: -${gridSize}, top: -${gridSize} etc.
+      if (quasi.value.raw.endsWith('-') && !t.isArrowFunctionExpression(nodeExpression)) {
+        quasi.value.raw = quasi.value.raw.substring(0, quasi.value.raw.length - 1);
```

Comment:
super nit:  I think `quasi.value.raw.slice(-1)` will also do the job.

Reply:
so close :)

`quasi.value.raw.slice(0, -1).trim()` did the trick